service-infrastructure/modules.tf

```diff
@@ -284,194 +371,113 @@ module "ecs_warehouse" {
   create_internal_alb           = false
 }
 
-module "rds_warehouse" {
+module "warehouse_database" {
   source = "./aurora_rds"
 
-  prefix                = "${local.prefix}-warehouse"
+  prefix                = "${local.prefix}-warehouse-database"
   db_name               = "epb"
   vpc_id                = module.networking.vpc_id
   subnet_group_name     = module.networking.private_subnet_group_name
-  security_group_ids    = [module.ecs_warehouse.ecs_security_group_id, module.bastion.security_group_id]
+  security_group_ids    = [module.warehouse_application.ecs_security_group_id, module.bastion.security_group_id]
   storage_backup_period = var.storage_backup_period
   instance_class        = "db.t3.medium"
 }
 
-module "redis_warehouse" {
+module "warehouse_redis" {
   source = "./elasticache"
 
-  prefix                        = "${local.prefix}-warehouse"
+  prefix                        = "${local.prefix}-warehouse-redis"
```

Comment:
This will create things like epb-intg-warehouse-redis-redis. Prefixes should be concise and per purpose. Here the purpose is warehouse on integration environment. This is shared with 'application' and if it exists 'database'

Reply:
This applies elsewhere as well

Reply:
Agree, have removed redis and database, to the prefix, where it will be later referenced by the resource.